src/containers/Sidebar/Sidebar.test.jsx

```diff
@@ -207,4 +211,43 @@ describe('<Sidebar />', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+
+  it('renders correctly when subscriptionManagementScreen is false', () => {
+    // should cause both subscription management and enrollment links to not be present
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        enableSubscriptionManagementScreen: false,
+      },
+    });
+
+    render(<SidebarWrapper store={store} />);
+    const subscriptionEnrollmentLink = screen.queryByRole('link', { name: 'Subscription Enrollment' });
+    const subscriptionManagementLink = screen.queryByRole('link', { name: 'Subscription Management' });
+    expect(subscriptionEnrollmentLink).toBeNull();
+    expect(subscriptionManagementLink).toBeNull();
+  });
+
+  it('renders correctly when subscriptionManagementScreen and Bulk Enrollment is enabled', () => {
+    // should cause both subscription management and enrollment links to not be present
+    const store = mockStore({
+      sidebar: {
+        ...initialState.sidebar,
+      },
+      portalConfiguration: {
+        enableSubscriptionManagementScreen: true,
+      },
+    });
+
+    features.BULK_ENROLLMENT = true;
```

Comment:
should this be true for all tests or?

Reply:
I had originally made it true for all tests, but then that just affects snapshot tests, so meh.

Reply:
Yeah I think we should eventually really limit if not remove all snapshot tests, really only useful for the smallest component boundaries I think